newui/content/index.html

```diff
@@ -0,0 +1,233 @@
+
+    
+
+<div class="masthead">
+  <div class="masthead__inner-wrap" id="masthead__inner-wrap">
+    <div class="masthead__menu">
+      <nav id="site-nav" class="greedy-nav">
+        
+          <a class="site-logo" href="/"><img src="/assets/images/hudi.png" alt=""></a>
+        
+        <a class="site-title" href="/">
+          Apache Hudi
+          <span class="site-subtitle">0.5.1-SNAPSHOT(incubating)</span>
+        </a>
+        <ul class="visible-links"><li class="masthead__menu-item">
+              <a href="/docs/quick-start-guide.html" target="_self" >Documentation</a>
+            </li><li class="masthead__menu-item">
+              <a href="/community.html" target="_self" >Community</a>
+            </li><li class="masthead__menu-item">
+              <a href="/roadmap.html" target="_self" >Roadmap</a>
+            </li><li class="masthead__menu-item">
+              <a href="/activity.html" target="_self" >Activities</a>
+            </li><li class="masthead__menu-item">
+              <a href="https://cwiki.apache.org/confluence/display/HUDI/FAQ" target="_blank" >FAQ</a>
+            </li><li class="masthead__menu-item">
+              <a href="/releases.html" target="_self" >Releases</a>
+            </li></ul>
+        <button class="greedy-nav__toggle hidden" type="button">
+          <span class="visually-hidden">Toggle menu</span>
+          <div class="navicon"></div>
+        </button>
+        <ul class="hidden-links hidden"></ul>
+      </nav>
+    </div>
+  </div>
+</div>
+
+
+    <div class="page__hero--overlay" style="background-color: #079fca !important;">
+      <div class="wrapper" style="background-size: 480px;background-repeat: no-repeat; background-position: right center;  background-image: url('/assets/images/logo-big.png');">
+        <h1 id="page-title" class="page__title" itemprop="headline">
+          Welcome to Apache Hudi
+        </h1>
+
+        <p class="page__lead">Apache Hudi Stands for Hadoop Upserts and Incrementals to manage the storage of large analytical datasets on HDFS.<br /> <small><a href="https://github.com/apache/incubator-hudi/releases/tag/release-0.5.0-incubating" target="_blank">Latest release 0.5.0-incubating</a></small>
+</p>
+        <p>
+          <a href="/docs/quick-start-guide.html" class="btn btn--light-outline btn--large"><i class="fa fa-paper-plane"></i> Get Starting</a>
+        </p>
+      </div>
+    </div>
+
+    <div class="page__hero--overlay">
+      <div class="wrapper">
+        <h1 style="font-size: 1.953em">
+          Hudi Data Lakes
+        </h1>
+        <div class="archive__item-excerpt" style="font-size: 0.9em">
+          <p>Hudi brings stream processing to big data, providing fresh data while being an order of magnitude efficient over traditional batch processing.</p>
+        </div>
+        <div class="archive__item-teaser">
+          <img src="/assets/images/hudi-delta-lake.png" style="display:block;margin-left:auto;margin-right:auto;width: auto">
```

Comment:
hudi-delta-lake? please rename this file

Reply:
Also this image is not probably the best/accurate thing out there.. can you remove this section for now.. and file another JIRA to add it in later.. @bhasudha is already tracking the landing page image redesign I think.

Reply:
I think keep it until the new image created.